content/cn/use_cases.html

```diff
@@ -341,24 +341,24 @@ <h2 id="近实时摄取">近实时摄取</h2>
 <p>将外部源(如事件日志、数据库、外部源)的数据摄取到<a href="http://martinfowler.com/bliki/DataLake.html">Hadoop数据湖</a>是一个众所周知的问题。
 尽管这些数据对整个组织来说是最有价值的，但不幸的是，在大多数(如果不是全部)Hadoop部署中都使用零散的方式解决，即使用多个不同的摄取工具。</p>
 
-<p>对于RDBMS摄取，Hudi提供__通过更新插入达到更快加载__，而不是昂贵且低效的批量加载。例如，您可以读取MySQL BIN日志或<a href="https://sqoop.apache.org/docs/1.4.2/SqoopUserGuide.html#_incremental_imports">Sqoop增量导入</a>并将其应用于
+<p>对于RDBMS摄取，Hudi提供<strong>通过Upserts提供了更快加载</strong>，而不是昂贵且低效的批量加载。例如，您可以读取MySQL Binlog或<a href="https://sqoop.apache.org/docs/1.4.2/SqoopUserGuide.html#_incremental_imports">Sqoop增量导入</a>并将其应用于
```

Comment:
Does <strong> make sense? and correspond to the English document?

Reply:
In markdown, we use `__` as bold, but we should use `<strong> </ strong>` tags in html.